reference/MarkdownViewer/main.cpp

```diff
@@ -0,0 +1,73 @@
+#include <CtrlLib/CtrlLib.h>
+
+#include <plugin/md/Markdown.h>
+#include <plugin/bmp/bmp.h>
+#include <plugin/jpg/jpg.h>
+
+using namespace Upp;
+
+struct MarkdownViewer : TopWindow {
+	typedef MarkdownViewer CLASSNAME;

```

Comment:
This is not needed. Just use lambda callback instead. It was deprecated some time ago...

Reply:
Done.